src/fdo_magic/builtin/runtime.rs

```diff
@@ -12,13 +13,10 @@ use crate::fdo_magic::ruleset;
 use crate::Mime;
 
 fn search_paths(filename: &str) -> Vec<PathBuf> {
-    let mut search_paths = vec![
-        PathBuf::from("/usr/share/mime").join(filename),
-        PathBuf::from("/usr/local/share/mime").join(filename),
-        PathBuf::from("/opt/homebrew/share/mime").join(filename),
-    ];
-    if let Some(home) = home::home_dir() {
-        search_paths.push(home.join(".local/share/mime").join(filename));
+    let mut search_paths: Vec<PathBuf> = Vec::new();
+    let dirs = env::var("XDG_DATA_DIRS").unwrap_or_default();
```

Comment:
In addition to `$XDG_DATA_DIRS`, please search `$XDG_DATA_HOME` as well.

Please implement the fallback behavior described in the XDG Base Directory Specification:

> If `$XDG_DATA_HOME` is either not set or empty, a default equal to `$HOME/.local/share` should be used.

>  If `$XDG_DATA_DIRS` is either not set or empty, a value equal to `/usr/local/share/:/usr/share/` should be used.

In addition, I expect we should keep the "/opt/homebrew/share" fallback on macOS.

Reply:
considering that the XDG specification isn't used on windows anyway my updated implementation should be fine as all UNIX-like operating systems set `$HOME`. That being said maybe the entire xdg-based runtime loading should be disabled for windows as a target os and a windows specific impl could be added using `HKEY_LOCAL_MACHINE\SOFTWARE\Classes\MIME\Database` (if somebody cares about windows support, i certainly don't)